packages/mdc-switch/mdc-switch.scss

```diff
@@ -45,79 +38,86 @@
       transform: translateX($mdc-switch-knob-active-margin);
     }
   }
+}
+
+.mdc-switch__background {
+  display: block;
+  position: relative;
+  width: $mdc-switch-track-width;
+  height: $mdc-switch-track-height;
+  border-radius: 7px;
```

Comment:
Once this is merged, can you create a follow up PR that changes _**all**_ the `border-radius: Npx` properties in this file to `border-radius: 50%`? AFAIK there's no need for specific `px` values.